.readthedocs.yml

```diff
@@ -22,11 +22,8 @@ python:
   version: 3.8
   install:
     - requirements: doc/requirements.txt
-    - requirements: qchem/requirements.txt
```

Comment:
@soranjh @josh146 this is the culprit for the docs build, `qchem/requirements.txt` was implicitly installing `networkx` and `retworkx`. Should be a good fix to add them :+1:

Reply:
Ah!! Thanks @antalszava this is a great find. @soranjh so it should be sufficient to simply add
```
networkx
retworkx
```
to `doc/requirements.txt`.